tests/component/test_stream_readers.py

```diff
@@ -1,47 +1,110 @@
 import ctypes
 import math
+from typing import List
 
 import numpy
 import pytest
 
 import nidaqmx
-from nidaqmx.stream_readers import AnalogMultiChannelReader, AnalogSingleChannelReader
+from nidaqmx.stream_readers import (
+    AnalogMultiChannelReader,
+    AnalogSingleChannelReader,
+    AnalogUnscaledReader,
+    PowerBinaryReader,
+    PowerMultiChannelReader,
+    PowerSingleChannelReader,
+)
+
+VOLTAGE_EPSILON = 1e-3
+VOLTAGE_BINARY_EPSILON = 4
```

Comment:
This is `_volts_to_codes` of the previous value rounded up. Make the code do that.